src/style.css

```
@@ -0,0 +1,107 @@
+@import url(../src/stars--2.css);
+@import url(../src/stars--3.css);
+@import url(../src/stars--4.css);
+@import url(../src/stars--5.css);
+body {
+  margin: 0;
+  margin-top: 2px;
+  margin-left: 20px;
+  padding: 0;
+  width: 116px;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars--0 .stars__star {
+  background-image: url(../src/images/star.svg);
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  margin-right: 4px;
+}
+
+.stars--0 .stars__star:last-child {
+  margin-right: 0;
+}
```

```suggestion
 .stars__star:last-child {
  margin-right: 0;
}
```

no need to use modifiers to remove margin-right from the last stars. You want to remove this indent from all last stars in a row